python-frontend/src/test/java/org/sonar/python/types/InferredTypesTest.java

```diff
@@ -379,6 +459,10 @@ void test_type_from_protobuf() throws TextFormat.ParseException {
       "fully_qualified_name: \"mod.t\""
       ))
       .isEqualTo(InferredTypes.or(STR, INT));
+    assertThat(protobufType(
+      "kind: INSTANCE\n" +
+      "fully_qualified_name: \"typing._SpecialForm\""
+      )).isEqualTo(anyType());
```

Comment:
Sorry about this as well. This is not related to the PR. It just raises the coverage.